packages/frontend/amp/components/Onward.tsx

```diff
@@ -147,10 +147,10 @@ export const Onward: React.FC<{
     );
 
     // Outbrain is compliant if it appears in the top 2 containers
-    const outbrainIsCompliant = storyPackage.concat(series).length > 1;
+    const outbrainIsNotCompliant = storyPackage.concat(series).length > 1;
     const outbrain = shouldHideAds
         ? []
-        : [outbrainContainer(webURL, outbrainIsCompliant)];
+        : [outbrainContainer(webURL, !outbrainIsNotCompliant)];
```

Comment:
It might be better to avoid the `Not` (falsy) const naming/value and then immediately negate it. As that kind of construct is much harder to read/easier to get wrong. I.e. instead just fix the initial test condition:

    const outbrainIsCompliant = storyPackage.concat(series).length <= 1;
 
?

Reply:
Yes. I said I went for the tiniest change. Let me change the variable name :)